src/StoreApi/Schemas/V1/OrderSchema.php

```diff
@@ -0,0 +1,161 @@
+							'type'        => 'mixed',
+							'context'     => [ 'view', 'edit' ],
+						],
+						'quantity'  => [
+							'description' => __( 'Quantity ordered.', 'woo-gutenberg-products-block' ),
+							'type'        => 'integer',
+							'context'     => [ 'view', 'edit' ],
+						],
+						'subtotal'  => [
+							'description' => __( 'Line subtotal (before discounts).', 'woo-gutenberg-products-block' ),
+							'type'        => 'string',
+							'context'     => [ 'view', 'edit' ],
+						],
+						'meta_data' => [
+							'description' => __( 'Meta data.', 'woo-gutenberg-products-block' ),
+							'type'        => 'array',
+							'context'     => [ 'view', 'edit' ],
+							'items'       => [
+								'type'       => 'object',
+								'properties' => [
+									'id'            => [
+										'description' => __( 'Meta ID.', 'woo-gutenberg-products-block' ),
+										'type'        => 'integer',
+										'context'     => [ 'view', 'edit' ],
+										'readonly'    => true,
+									],
+									'key'           => [
+										'description' => __( 'Meta key.', 'woo-gutenberg-products-block' ),
+										'type'        => 'string',
+										'context'     => [ 'view', 'edit' ],
+									],
+									'value'         => [
+										'description' => __( 'Meta value.', 'woo-gutenberg-products-block' ),
+										'type'        => 'mixed',
+										'context'     => [ 'view', 'edit' ],
+									],
+									'display_key'   => [
+										'description' => __( 'Meta key for UI display.', 'woo-gutenberg-products-block' ),
+										'type'        => 'string',
+										'context'     => [ 'view', 'edit' ],
+									],
+									'display_value' => [
+										'description' => __( 'Meta value for UI display.', 'woo-gutenberg-products-block' ),
+										'type'        => 'string',
+										'context'     => [ 'view', 'edit' ],
+									],
+								],
+							],
+						],
+					],
+				],
+			],
+			'totals'     => [
+				'description' => __( 'Order totals.', 'woo-gutenberg-products-block' ),
+				'type'        => 'object',
+				'context'     => [ 'view', 'edit' ],
+				'items'       => [
+					'type'       => 'object',
+					'properties' => [
+						'label' => [
+							'description' => __( 'Label.', 'woo-gutenberg-products-block' ),
+							'type'        => 'string',
+							'context'     => [ 'view', 'edit' ],
+							'readonly'    => true,
+						],
+						'value' => [
+							'description' => __( 'Value.', 'woo-gutenberg-products-block' ),
+							'type'        => 'string',
+							'context'     => [ 'view', 'edit' ],
+							'readonly'    => true,
+						],
+					],
+				],
+			],
+		];
+	}
+
+	/**
+	 * Get an order for response.
+	 *
+	 * @param \WC_Order $order Order instance.
+	 * @return array
+	 */
+	public function get_item_response( $order ) {
+		return [
+			'id'         => $order->get_id(),
+			'line_items' => $this->get_item_data( $order ),
+			'totals'     => $order->get_order_item_totals(),
```

Comment:
I initially manually added all the totals here.  The `get_order_item_totals` method includes HTML and currencies so we'll need to dangerously set and sanitize in React, but it uses more filters from WordPress which could be useful.

I'm not sure which one is better.

Reply:
Thanks for the comment, @senadir.  I'm not sure that I understand your suggestion here.  

>endpoint response should be consisent with the rest of requsts, so items instead of line items and it should be formated via CartItemSchema, same for totals...

It seems like the solution we decided not to go with.  That solution was to match order data to the cart schema so we could use the existing cart block.  Did I understand your suggestion correctly?

After this PR adds the pay-for-order order endpoint, [we will create a `CheckoutOrderPayForOrderBlock` to hydrate the order using this endpoint.](https://href.li/?https://github.com/Automattic/woocommerce-blocks/blob/b1e07428a8858f7c76c41bc15c87d23f39cb01d5/src/BlockTypes/Checkout.php#L493)  That's why we don't convert the order schema to the cart item schema here.